assets/scripts/lobby/options.js

```diff
@@ -1176,18 +1176,18 @@ $(document).ready(() => {
   // On first run, update the colours
 
   for (let i = 0; i < 10; i++) {
-    if (!docCookies.hasItem(`player${i}HighlightColour`)) {
-      docCookies.setItem(
+    if (!localStorage.getItem(`player${i}HighlightColour`)) {
+      localStorage.setItem(
         `player${i}HighlightColour`,
         defaultColours[i],
         Infinity
       );
     }
     $(`#player${i}HighlightColour`)[0].jscolor.fromString(
-      docCookies.getItem(`player${i}HighlightColour`)
+      localStorage.getItem(`player${i}HighlightColour`)
     );
     $(`#player${i}HighlightColour2`)[0].jscolor.fromString(
-      docCookies.getItem(`player${i}HighlightColour`)
+      localStorage.getItem(`player${i}HighlightColour`)
```

Comment:
Can you make a separate file called `cookies.js` and write two function: 
`StorePlayerHighlightColour(number, colour)`
`GetPlayerHighlightColour(number)`

And then use it in these other files?